src/main/java/com/github/dockerjava/api/model/LogConfig.java

```diff
@@ -0,0 +1,36 @@
+package com.github.dockerjava.api.model;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class LogConfig {
+    
+    @JsonProperty("Type")
+    public String type = "json-file";
+    
+    @JsonProperty("Config")
+    public List<String> config = null;
```

Comment:
It should be noted this variable does nothing through docker at the moment (and I vaguely remember talk of it being removed) and serves only to help tests run successfully.